sql/database.sql

```diff
@@ -298,6 +298,8 @@ INSERT INTO `categories` VALUES (26, 'Drawings - Eye', '', 17, 47, 48, 'patients
 INSERT INTO `categories` VALUES (27, 'Onsite Portal', '', 1, 51, 56, 'patients|docs');
 INSERT INTO `categories` VALUES (28, 'Patient', '', 27, 52, 53, 'patients|docs');
 INSERT INTO `categories` VALUES (29, 'Reviewed', '', 27, 54, 55, 'patients|docs');
+-- @bradymiller is this the right aco for this?  We really don't want people with patient stuff to have access to this doc
+INSERT INTO `categories` VALUES (30, 'FHIR Export Document', '', 1, 55, 56, 'admin|super');
```

Comment:
```
INSERT INTO `categories` VALUES (30, 'FHIR Export Document', '', 1, 57, 58, 'admin|super');
```
And the number in the categories entry needs to be bumped from 57 to 59

Reply:
also, the aco looks good

Reply:
K should be fixed now.

Reply:
dang, i almost forgot, we also need to change following line in database.sql :)
```
INSERT INTO `categories_seq` VALUES (29);
```
to:
```
INSERT INTO `categories_seq` VALUES (30);
```